common/maplib/map.mjs

```diff
@@ -22,6 +22,14 @@ class GameMap {
     }
   }
 
+  /**
+   * Return the GraphicAsset that is in use with this class.
+   * @return {GraphicAsset} asset
+   */
+  graphicAsset() {
```

Comment:
I think one would use `GameMap.myasset` instead of `GameMap.graphicAsset()` if he wants to access the asset.
Maybe we should rename `this.myasset` to `this.graphicAsset`.
Also, `this.mymap` should also be renamed.